x11/xcb/Portfile

```diff
@@ -6,6 +6,7 @@ revision         3
 categories       x11
 maintainers      gmail.com:qbarnes
```

Comment:
please add tour GitHub handle here as well, and, ideally, also "openmaintainer".